_studio/shared/asc/include/cpu_detect.h

```diff
@@ -32,11 +32,11 @@
 #pragma warning(disable:4505)
 #endif
 
-static mfxI32 CpuFeature_SSE41() {
+static inline mfxI32 CpuFeature_SSE41() {
     return((__builtin_cpu_supports("sse4.1")));
 }
 
-static mfxI32 CpuFeature_AVX2() {
+static inline mfxI32 CpuFeature_AVX2() {
```

Comment:
Fix warning "Wunused-function". Both functions are one-line-functions which use 3 times in _studio/shared/asc/src/asc.cpp

Reply:
In header file this functions do not wrapped in any macros and use in _studio/shared/asc/src/asc.cpp.